bin/install_uninstall.ml

```diff
@@ -46,9 +46,13 @@ let get_dirs context ~prefix_from_command_line ~from_command_line =
     match v with
     | Some v -> v
     | None ->
+      (* We suggest that the user sets --prefix first rather than the specific
+         missing option, since this is the most common case. *)
```

Comment:
```suggestion
      (* We suggest that the user sets --prefix first rather than the specific
         missing option, since this is the most common case. *)
```

I'd remove the comment but I'm fine with it if you'd like to keep it.